<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_spectrometer_parameters_dialog</class>
 <widget class="QDialog" name="set_spectrometer_parameters_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrometer parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="heading">
       <property name="text">
        <string>Set spectrometer parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="integration_time_layout">
       <item>
        <widget class="QLabel" name="integration_time_label">
         <property name="text">
          <string>Integration time (ms)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="integration_time_spinbox">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="scans_average_layout">
       <item>
        <widget class="QLabel" name="scans_average_label">
         <property name="text">
          <string>Scans to average</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scans_average_spinbox">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="electrical_dark_checkbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Correct for electrical dark</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="substract_background_checkbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Substract background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filter_checkbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Filter from</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="filter_lower_limit_spinbox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="to_label">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="filter_upper_limit_spinbox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <item>
          <property name="text">
           <string>nm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cm^-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>eV</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>set_spectrometer_parameters_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>set_spectrometer_parameters_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
